ethereum/contracts/Starport.sol

```diff
@@ -87,8 +89,8 @@ contract Starport {
      * @dev Use `lock` to lock CASH or collateral assets.
      */
     function lockEthTo(string memory chain, bytes32 recipient) public payable {
-        require(address(this).balance <= supplyCaps[ETH_ADDRESS], "Supply Cap Exceeded");
-        emit Lock(ETH_ADDRESS, msg.sender, chain, recipient, msg.value);
+        require(address(this).balance <= supplyCaps[NATIVE_TOKEN_ADDRESS], "Supply Cap Exceeded");
+        emit Lock(NATIVE_TOKEN_ADDRESS, msg.sender, chain, recipient, msg.value);
```

Comment:
Not sure its worth logging the `NATIVE_TOKEN_ADDRESS` every time?

Reply:
yeah i think this just makes lock events generic